src/main/java/com/faforever/client/remote/MockFafServerAccessor.java

```diff
@@ -245,17 +234,17 @@ public void requestMatchmakerInfo() {
   }
 
   @Override
-  public CompletableFuture<GameLaunchMessage> startSearchLadder1v1(Faction faction) {
-    logger.debug("Searching 1v1 match with faction: {}", faction);
+  public CompletableFuture<GameLaunchMessage> startSearchMatchmaker() {
+    logger.debug("Starting matchmaker game");
     GameLaunchMessage gameLaunchMessage = new GameLaunchMessage();
     gameLaunchMessage.setUid(123);
-    gameLaunchMessage.setMod(KnownFeaturedMod.DEFAULT.getTechnicalName());
+    gameLaunchMessage.setMod(KnownFeaturedMod.DEFAULT.getTechnicalName()); // TODO
```

Comment:
TODO what also refer to an issue if you make a TODO

Reply:
I am not sure, I guess it refers to changing the gamemode to a matchmaking mode. However we use FAF gamemode, so I guess this todo is obsolete.